sigs.yaml

```
@@ -761,15 +761,15 @@ sigs:
       time: "14:30"
       tz: UTC
       frequency: monthly
-      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+      url: https://zoom.us/j/586836662
       archive_url: https://docs.google.com/document/d/1SpxvmOgHDhnA72Z0lbhBffrfe9inQxZkU9xqlafOW9k/edit
       recordings_url: https://www.youtube.com/watch?v=yQLeUKi_dwg&list=PL69nYSiGNLP2JNdHwB8GxRs2mikK7zyc4
     - description: Azure Subproject Meeting (Third Tuesday UTC, Monday Pacific)
       day: Tuesday
       time: "1:00"
       tz: UTC
       frequency: monthly
-      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+      url: no-zoom-link
```

```suggestion
      url: https://zoom.us/j/586836662
```
It looks like they use the same zoom link for both 👍